[2023-07-09T08:42:47.181+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-07-09T08:42:35.257664+00:00 [queued]>
[2023-07-09T08:42:47.195+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_data manual__2023-07-09T08:42:35.257664+00:00 [queued]>
[2023-07-09T08:42:47.196+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-09T08:42:47.225+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_data> on 2023-07-09 08:42:35.257664+00:00
[2023-07-09T08:42:47.234+0000] {standard_task_runner.py:57} INFO - Started process 2922 to run task
[2023-07-09T08:42:47.237+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_data', 'manual__2023-07-09T08:42:35.257664+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpbtl3r8kk']
[2023-07-09T08:42:47.238+0000] {standard_task_runner.py:85} INFO - Job 118: Subtask load_data
[2023-07-09T08:42:47.303+0000] {task_command.py:410} INFO - Running <TaskInstance: etl_dag.load_data manual__2023-07-09T08:42:35.257664+00:00 [running]> on host 5865fe7b5e5b
[2023-07-09T08:42:47.542+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-07-09T08:42:35.257664+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-09T08:42:35.257664+00:00'
[2023-07-09T08:42:47.554+0000] {base.py:73} INFO - Using connection ID 'potsqqq' for task execution.
[2023-07-09T08:42:47.638+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-07-09T08:42:47.666+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_data, execution_date=20230709T084235, start_date=20230709T084247, end_date=20230709T084247
[2023-07-09T08:42:47.731+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-09T08:42:47.755+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
